components/cli/cmd/cellery/list_components.go

```diff
@@ -48,7 +49,7 @@ func newListComponentsCommand() *cobra.Command {
 			return nil
 		},
 		Run: func(cmd *cobra.Command, args []string) {
-			commands.RunListComponents(args[0])
+			commands.RunListComponents(cli, args[0])
```

Comment:
Can there be any error when executing RunListComponents?

Reply:
Yes there can be, however RunListComponents is not refactored yet and therefore does not return an error as at now. Will update and return the error and handle it in here when RunListComponents refactoring is being done.